### ISSUE TITLE
How to enable the nRF51 BT dongle to work with mesh?

### ISSUE THREAD

**Issue by feri** [feri/Junction2018_Support#3]
The nRF51 dongle comes with a default application. It is not suitable for our needs at Junction. There is a simple process that makes the dongle usable. It is basically to build and flash the nRF51-PCA10028 sample Zephyr app onto the dongle and run btattach on the host where the dongle is attached.
We need a simple how-to that describes this procedure.

**Comment by kolodgrz** [comment]
Hi @feri did you tried to do it by yourself ? It shouldn't be so hard. I did it with nrf52. Currently I'm out of office so I don't have physically nrf51 but have nrf52840dk board. 

For sure it's needed:
1) flash zephyr_root/samples/bluetooth/hci_uart on board
2) have initial procedure which I've implemented locally - I could upload a simple patch here since Inga and Brian said that is no need with their setup. Without it our controller wouldn't be initialized to scan or even advertise - This is a basic must do procedure for LE HCI device.
3) Have BlueZ with btattach

**Comment by feri** [comment]
Thanks @kolodgrz! 
Yes, we did that with Johan, and I made notes on paper. It would be wonderful if meshd would take care of these, one less hurdle before getting started with app development. 
So let's see if we can close this issue even without writing down that procedure.

**Comment by kolodgrz** [comment]
@feri exactly. LE HCI device must be initialized to receive scan responses - without this setup any non initialized by some external "stacks" wouldn't be able to work. I don't know why this issue came so late... I've raised this issue long time ago and everyone ignored it :D

**Comment by kolodgrz** [comment]
To sumamrize @feri it means that you'll be doing this readme for this ?

**Comment by feri** [comment]
Yes, I will be, if needed. For Junction though we will have all hardware ready, Johan just flashed all the dongles today. But I guess it's a good info for someone who wants to get started and won't be at the hackathon.
